skflow/__init__.py

```diff
@@ -28,7 +28,7 @@
 from skflow.trainer import TensorFlowTrainer
 from skflow import models, data_feeder
 from skflow import preprocessing
-
+from skflow.io.pandas_io import *
```

Comment:
I am changing it to `from skflow.io import *`, feel free to change it later to something else.